### ISSUE TITLE
Supervised Metric Pretraining

### ISSUE THREAD

**Issue by chingisooinar-second** [microsoft/metric-transfer.pytorch#1]
Hello,
I loved reading your paper. However, I couldn't find a Supervised Metric Pretraining. Can I contribute by adding it?
Thank you

**Comment by bl0** [comment]
Hi, thanks a lot for your interest in our work. May I ask which supervised metric pre-training method you want to add?

**Comment by chingisooinar-second** [comment]
I am currently working with a labelled dataset having direct commonalities with my target task. In the paper, you mention an equation using which we can maximize the likelihood of each example being supported by other examples belonging to the same category. I couldn't find a sample code in the repository, so I was wondering if I can add that.

**Comment by bl0** [comment]
Thanks. You can try to submit a PR and I will check that.
